packages/terra-navigation-side-menu/src/NavigationSideMenu.jsx

```diff
@@ -306,17 +328,28 @@ class NavigationSideMenu extends Component {
       this.handleEvents(event, item, key);
     };
 
+    if (this.props.variant === 'drill-in' && key === 'empty-child-key') {
+      return <StatusView variant="no-data" />;
+    }
+
+    if (this.props.variant !== 'drill-in' && key === 'empty-child-key') {
+      return null;
+    }
+
     return (
       <MenuItem
         id={item.id}
-        hasChevron={item.hasSubMenu || (item.childKeys && item.childKeys.length > 0)}
+        hasChevron={item.hasSubMenu || (item.childKeys && item.childKeys.length >= 0)}
```

Comment:
Why '=' is added?

Reply:
To add chevron to item/folder with no items/documents

Reply:
So is it supposed to be added for both the variants?

Reply:
Reverted as no longer needed: https://github.com/cerner/terra-framework/pull/2161/commits/638d506e3d19e7a389d027232a52e61b0b0adfac